testing/smoke/released-artifacts/README.md

```diff
@@ -0,0 +1,21 @@
+# Smoke-test project for released artifacts
+
+This module can be used to smoke test the released artifacts and verify they are all properly deployed.
+
+## Steps for verifications
+
+### Update released versions
+
+Update the following properties in `pom.xml`:
+
+```
+core.components.version
+core.components.version.it-tests
+core.components.version.extension
+```
```

Comment:
I would rename as follows:
core.components.version.it-tests -> core.components.it-tests.version
core.components.version.extension -> core.components.extension.version